models/plays_by_pageview.sql

```diff
@@ -0,0 +1,140 @@
+{{ 
+  config(
+    materialized='table',
+    unique_key = 'play_id',
+    sort = 'start_tstamp',
+    dist = 'play_id'
+  )
+}}
+
+with prep as (
+
+  select
+    {{ dbt_utils.hash('i.page_view_id||i.media_id') }} play_id,
+    i.page_view_id,
+    i.media_id,
+    max(i.domain_sessionid) as domain_sessionid,
+    i.domain_userid,
+    i.duration,
+    i.title,
+    i.media_type,
+    i.media_player_type,
+    i.page_referrer,
+    i.content_url,
+    i.geo_region_name,
+    i.br_name,
+    i.dvce_type,
+    i.os_name,
+    i.os_timezone,
+    coalesce(sum(p.weight_rate * i.duration), 0)::int as play_time_sec_estimated,
+    min(start_tstamp) start_tstamp,
+    sum(case when i.event_type in ('seek', 'seeked') then 1 else 0 end) as seeks,
+    max(case 
```

Comment:
Not sure I understand the variable naming logic here, why is this `last_percent_progress`? Isn't it more reporting on most progress made?

Reply:
it refers to the last percent progress that was reached during the play within the page view but I can rename it to `max_percent_progress` to make it clearer